src/seedu/addressbook/Main.java

```diff
@@ -74,12 +76,25 @@ private void exit() {
         ui.showGoodbyeMessage();
         System.exit(0);
     }
+    
+    private void checkStorageFile() throws NoStorageFileException {
+    	File storageFile = new File(DEFAULT_STORAGE_FILEPATH);
+    	if(!storageFile.exists()) {
+    		throw new NoStorageFileException("Storage file cannot be found");
+    	}
+    }
 
     /** Reads the user command and executes it, until the user issues the exit command.  */
     private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            try {
+				checkStorageFile();
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.